web/pdf_page_view.js

```diff
@@ -84,6 +85,7 @@ import { XfaLayerBuilder } from "./xfa_layer_builder.js";
  *   the necessary layer-properties.
  * @property {boolean} [enableHWA] - Enables hardware acceleration for
  *   rendering. The default value is `false`.
+ * @property {boolean} [enableAutoLinking] - Enable automatic linking of URLs.
```

Comment:
This is perhaps not all that helpful if you don't already know what the feature does.
```suggestion
 * @property {boolean} [enableAutoLinking] - Enable creation of hyperlinks from
     text that look like URLs. The default value is `false`.
```